xmake/plugins/project/clang/compile_commands.lua

```diff
@@ -44,8 +44,10 @@ function _translate_arguments(arguments)
         if idx == 1 and is_host("windows") and path.extension(arg) == "" then
             arg = arg .. ".exe"
         end
-        if arg:find("-isystem", 1, true) then
```

Comment:
To fix the problem you describe, simply change this

```lua
        if arg == "-isystem" then
```

Reply:
Clang accept arguments like `-isystem<directory>`. Same for  `-isystem-after<directory>`. arg:startswith handles those cases